.gitignore

```diff
@@ -4,3 +4,5 @@ rpmbuild
 /image-builder-migrate-db
 /image-builder-migrate-db-tern
 dnf-json
+/.fleet
+/.idea
```

Comment:
Not too relevant but I am on JetBrains tools.

Reply:
Again, this might be better as a separate commit, but since it's pretty obvious what this does, I don't insist on it. :)